saleor/static/js/components/variantPicker/VariantPicker.js

```diff
@@ -52,6 +53,11 @@ export default class VariantPicker extends Component {
     this.matchVariantFromSelection();
   }
 
+  checkVariantAvailability = () => {
+    const { store } = this.props;
+    return store.variant.schemaData.availability === 'http://schema.org/InStock';
```

Comment:
This check shouldn't be based on `schemaData`, but on a relevant field from `store.variant`. If such availability field is not present there, it should be added in [the dictionary that is passed to variant picker](https://github.com/mirumee/saleor/blob/master/saleor/product/utils.py#L175).